modules/tools/src/main/java11/org/apache/ignite/tools/javadoc/IgniteLinkTaglet.java

```diff
@@ -69,65 +67,59 @@
         return true;
     }
 
-    /**
-     * Register this Taglet.
-     *
-     * @param tagletMap the map to register this tag to.
-     */
-    public static void register(Map<String, IgniteLinkTaglet> tagletMap) {
-        IgniteLinkTaglet tag = new IgniteLinkTaglet();
-
-        Taglet t = tagletMap.get(tag.getName());
-
-        if (t != null)
-            tagletMap.remove(tag.getName());
-
-        tagletMap.put(tag.getName(), tag);
-    }
-
     /**
      * Given the <code>Tag</code> representation of this custom tag, return its string representation.
```

Comment:
Looks like this "Given the <code>Tag</code>" should be a " Given the <code>DocTree</code>"